metadata.rb

```diff
@@ -8,8 +8,7 @@
 version '3.0.25'
 
 depends 'apache2'
-depends 'application'
-depends 'application_php'
+depends 'application', '~> 4.1.6'
```

Comment:
if removing application_php allows us to upgrade, why do we need to pin?  also, why are we pinning to such a specific version?